### ISSUE TITLE
Any setting to make flycheck errors throw backtraces to toggle-debug-on-error?

### ISSUE THREAD

**Issue by codygman** [flycheck/flycheck#1637]
I purposefully remove read permissions from Blah.hs and flycheck gives this log in *Messages*:

```
<no location info>: error:
    can't find file: /var/folders/0t/cbkrycvj6b3fbrpq0z2ftt380000gn/T/flycheck3nLus4/Blah.hs
```

I want to trace down the piece of code that errors out on this. I tried searching for the error string and couldn't find anything.

I typically just `M-x toggle-debug-on-error` with most elisp code, but it looks like flycheck is catching these errors so they're never really thrown and I don't get a traceback.

I'm getting an error about the temporary file location written to and I know why I'm getting it. What I want to know is how to find the code throwing the error when flycheck catches errors and I cannot rely on `toggle-debug-on-error` to lead me to that code.

Is there some flycheck debug settings? I don't see anything when I use apropros on `flycheck*debug`.

**Comment by cpitclaudel** [comment]
Are you sure this is an error thrown by Flycheck? The format makes it look like an error returned by an error checker.

**Comment by cpitclaudel** [comment]
Closing this for inactivity; please reopen if you want to continue this discussion.
